sw/host/opentitanlib/src/transport/chip_whisperer/mod.rs

```diff
@@ -65,26 +69,29 @@ impl<B: Board> ChipWhisperer<B> {
     // Initialize the IO direction of some basic pins on the board.
     fn init_pin_directions(&self) -> anyhow::Result<()> {
         let device = self.device.borrow();
-        device.pin_set_output(B::PIN_TRST, true)?;
-        device.pin_set_output(B::PIN_POR_N, true)?;
-        device.pin_set_output(B::PIN_TAP_STRAP0, true)?;
-        device.pin_set_output(B::PIN_TAP_STRAP1, true)?;
-        device.pin_set_output(B::PIN_SW_STRAP0, true)?;
-        device.pin_set_output(B::PIN_SW_STRAP1, true)?;
-        device.pin_set_output(B::PIN_SW_STRAP2, true)?;
+
+        if let Ok(pins) = self.pin_mapper.list_gpio_pin_alias() {
+            for pin in pins.iter() {
+                device.pin_set_output(&pin, true)?;
+            }
+        }
```

Comment:
We should look at the pin config to get the list of pins, not the list of aliases.